build.gradle

```diff
@@ -145,6 +150,27 @@ task wrapper(type: Wrapper) {
     gradleVersion = '3.0'
 }
 
+//Start section for external jacoco test report
+tasks.withType(JavaCompile) {
+    if (System.getProperty("DEBUG", "false") == "true") {
```

Comment:
Maybe it would be better to read this property once and assign to some global variable